x-pack/test/apm_api_integration/tests/transactions/transactions_groups_detailed_statistics.spec.ts

```diff
@@ -127,53 +166,104 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             transactionsStatistics.currentPeriod[transactionNames[0]];
-          const metricsCurrentPeriod = metricsStatistics.currentPeriod[transactionNames[0]];
+          const metricsOneMinPeriod = metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsTenMinPeriod = metricsStatisticsTenMinute.currentPeriod[transactionNames[0]];
+          const metricsSixtyMinPeriod =
+            metricsStatisticsSixtyMinute.currentPeriod[transactionNames[0]];
+
           const transactionsThroughputMean = roundNumber(
             meanBy(transactionsCurrentPeriod.throughput, 'y')
           );
-          const metricsThroughputMean = roundNumber(meanBy(metricsCurrentPeriod.throughput, 'y'));
-          [transactionsThroughputMean, metricsThroughputMean].forEach((value) =>
+          const metricsOneMinThroughputMean = roundNumber(
+            meanBy(metricsOneMinPeriod.throughput, 'y')
+          );
+          const metricsTenMinThroughputMean = roundNumber(
+            meanBy(metricsTenMinPeriod.throughput, 'y')
+          );
+          const metricsSixtyMinThroughputMean = roundNumber(
+            meanBy(metricsSixtyMinPeriod.throughput, 'y')
+          );
+
+          expect(metricsTenMinThroughputMean).to.be.equal(
+            roundNumber(10 * (GO_PROD_RATE + GO_PROD_ERROR_RATE))
+          );
+          expect(metricsSixtyMinThroughputMean).to.be.equal(
+            roundNumber(60 * (GO_PROD_RATE + GO_PROD_ERROR_RATE))
+          );
+
+          [transactionsThroughputMean, metricsOneMinThroughputMean].forEach((value) =>
             expect(value).to.be.equal(roundNumber(GO_PROD_RATE + GO_PROD_ERROR_RATE))
           );
         });
 
         it('has same impact value for metrics and transactions data', () => {
           const transactionsCurrentPeriod =
             transactionsStatistics.currentPeriod[transactionNames[0]];
-          const metricsCurrentPeriod = metricsStatistics.currentPeriod[transactionNames[0]];
+          const metricsOneMinPeriod = metricsStatisticsOneMinute.currentPeriod[transactionNames[0]];
+          const metricsTenMinPeriod = metricsStatisticsTenMinute.currentPeriod[transactionNames[0]];
+          const metricsSixtyMinPeriod =
+            metricsStatisticsSixtyMinute.currentPeriod[transactionNames[0]];
 
           const transactionsImpact = transactionsCurrentPeriod.impact;
-          const metricsImpact = metricsCurrentPeriod.impact;
-          [transactionsImpact, metricsImpact].forEach((value) => expect(value).to.be.equal(100));
+          const metricsOneMinImpact = metricsOneMinPeriod.impact;
+          const metricsTenMinImpact = metricsTenMinPeriod.impact;
+          const metricsSixtyMinImpact = metricsSixtyMinPeriod.impact;
+
+          [
+            transactionsImpact,
+            metricsOneMinImpact,
+            metricsTenMinImpact,
+            metricsSixtyMinImpact,
+          ].forEach((value) => expect(value).to.be.equal(100));
```

Comment:
nit: we probably don't need the reassignments:
```suggestion
          [
            transactionsCurrentPeriod.impact,
            metricsOneMinPeriod.impact,
            metricsTenMinPeriod.impact,
            metricsSixtyMinPeriod.impact,
          ].forEach((value) => expect(value).to.be.equal(100));
```

Reply:
good catch, copy paste mistake